order/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+
+
+class Order(models.Model):
+  user          = models.ForeignKey('user.User',   on_delete = models.CASCADE)
+  order_status  = models.ForeignKey('OrderStatus', on_delete = models.CASCADE)
+  create_at     = models.DateField(auto_now_add = True)
+  update_at     = models.DateField(auto_now     = True)
+  
+
+  class Meta:
+    db_table = 'orders'
+
+
+class OrderStatus(models.Model):
+  order_id  = models.IntegerField(default = 0)
+  status    = models.CharField(max_length = 45)
+
+  class Meta:
+    db_table = 'order_status'
+
+
+class OrderItem(models.Model):
+  deliver     = models.ForeignKey('user.Address',    on_delete = models.CASCADE)
```

Comment:
field명은 Class명과 동일하게 수정해주세요~
```suggestion
  address     = models.ForeignKey('user.Address',    on_delete = models.CASCADE)
```

Reply:
수정 완료했습니다!! 감사합니다 🙇🏻‍♂️